### PR TITLE
Ins 3221 transfer to deposit

### PR DESCRIPTION
**Pull request by pol9kov**
…r third confirm

**- What I did**
Transfer from account to deposit
**- How I did it**
Transfer from migration daemon account to deposit after third migration; changed TestMigrationToken; + some refactor;
**- How to verify it**
TestMigrationToken
**- Description for the changelog**

### CODE REVIEW

File: logicrunner/builtin/foundation/helpers.go [hunk 1]
```diff
@@ -24,6 +24,8 @@ import (
 	"github.com/insolar/insolar/insolar/genesisrefs"
 )
 
+const XNS = "XNS"
```
Comment (LinnikD) [inline logicrunner/builtin/foundation/helpers.go]: its not a part of a foundation. move it back to contracts

File: logicrunner/builtin/contract/wallet/wallet.go [hunk 2]
```diff
@@ -42,7 +42,7 @@ func New(accountReference insolar.Reference) (*Wallet, error) {
 	}
 	accounts := make(foundation.StableMap)
 	// TODO: Think about creating of new types of assets and initial balance
-	accounts["XNS"] = accountReference.String()
```
Comment (LinnikD) [inline logicrunner/builtin/contract/wallet/wallet.go]: make "XNS" a constant

File: logicrunner/builtin/contract/wallet/wallet.go [hunk 3]
```diff
@@ -114,12 +107,12 @@ func (w *Wallet) Transfer(rootDomainRef insolar.Reference, assetName string, amo
...
 	}
 	acc := account.GetObject(*accRef)
-	err = acc.Transfer(amountStr, toAccount)
+	err = acc.TransferToAccount(amountStr, *toAccount)
 	if err != nil {
 		return nil, fmt.Errorf("failed to transfer: %s", err.Error())
 	}
 
-	feeWalletRef, err := cc.GetFeeWalletRef()
+	feeWalletRef, err := cc.GetFeeAccountRef()
```
[the remark this answers is not in the document]
Comment (megge-dream) [reply logicrunner/builtin/contract/wallet/wallet.go]: below you do `wallet.GetObject(feeWalletRef)`, but `feeWalletRef` is account ref now

File: logicrunner/builtin/contract/wallet/wallet.go (hunk 2 above)
Comment (pol9kov) [reply logicrunner/builtin/contract/wallet/wallet.go]: I made a constant for it yet

File: functest/deposit_migration_test.go [hunk 4]
```diff
@@ -56,6 +61,13 @@ func TestMigrationToken(t *testing.T) {
 	require.Equal(t, confirmerReferences[0], migrationDaemons[0].ref)
 	require.Equal(t, confirmerReferences[1], migrationDaemons[1].ref)
 	require.Equal(t, confirmerReferences[2], migrationDaemons[2].ref)
+
+	require.Equal(t, deposit["balance"], "1000")
```
[the remark this answers is not in the document]
Comment (pol9kov) [reply functest/deposit_migration_test.go]: Thank you

File: logicrunner/builtin/contract/wallet/wallet.go (hunk 3 above)
[the remark this answers is not in the document]
Comment (pol9kov) [reply logicrunner/builtin/contract/wallet/wallet.go]: Thank you

**Pull request merged**